test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertFrom-StringData.Tests.ps1

```diff
@@ -65,3 +65,50 @@ bazz = 2
 	$(ConvertFrom-StringData -StringData $sampleData).Values | Should -BeIn @("0","1","2")
     }
 }
+
+Describe "Delimiter parameter tests" -Tags "CI" {
+
+    $sampleData = @"
```

Comment:
It is used only in one test. Please move to the test.

Reply:
Done!